skyportal_fink_client/utils/skyportal_api.py

```diff
@@ -961,24 +969,35 @@ def get_taxonomy_id_including_classification(classification: str, url: str, toke
     ----------
         status_code : int
             HTTP status code
-        exists : bool
-            True if the classification is in the hierarchy, False otherwise
+        exists : string
```

Comment:
Nit: the name `exists` does not really reflect the meaning of the variable here (exists would rather be a boolean).

Reply:
Fixed ! I replaced the names of return arguments that dont have a declared variable (given name) by `out` as you suggested me to earlier.